contrib/coreos/user-data.example

```diff
@@ -1,25 +1,57 @@
 #cloud-config
 ---
 coreos:
-  etcd:
-    # generate a new token for each unique cluster from https://discovery.etcd.io/new
-    # uncomment the following line and replace it with your discovery URL
-    # discovery: https://discovery.etcd.io/12345693838asdfasfadf13939923
-    addr: $private_ipv4:4001
-    peer-addr: $private_ipv4:7001
-    # give etcd more time if it's under heavy load - prevent leader election thrashing
-    peer-election-timeout: 2000
-    # heartbeat interval should ideally be 1/4 or 1/5 of peer election timeout
-    peer-heartbeat-interval: 500
   fleet:
     # We have to set the public_ip here so this works on Vagrant -- otherwise, Vagrant VMs
     # will all publish the same private IP. This is harmless for cloud providers.
     public-ip: $private_ipv4
     # allow etcd to slow down at times
     etcd_request_timeout: 3.0
+    etcd_servers: http://127.0.0.1:2379
   units:
   - name: etcd.service
+    mask: true
```

Comment:
I am very wrong, and quite happy about it. This does seem to work just fine on current CoreOS 647.2.0:
```console
core@deis-01 ~ $ ps aux | grep etcd
etcd      1064  0.0  0.6 212924 13216 ?        Ssl  19:22   0:00 /usr/bin/docker run --net=host --rm --volume=/var/lib/etcd2:/var/lib/etcd2 --volume=/usr/share/ca-certificates:/etc/ssl/certs:ro -p 4001:4001 -p 2380:2380 -p 2379:2379 -p 7001:7001 --name etcd2 quay.io/coreos/etcd:v2.0.13 -name 59b8789c69504c34bb93c2c1b18c4765 -data-dir /var/lib/etcd2 -advertise-client-urls http://172.17.8.100:2379,http://172.17.8.100:4001 -listen-client-urls http://0.0.0.0:2379,http://0.0.0.0:4001 -initial-advertise-peer-urls http://172.17.8.100:2380,http://172.17.8.100:7001 -listen-peer-urls http://0.0.0.0:2380,http://0.0.0.0:7001 --heartbeat-interval 400 --election-timeout 2000 --discovery https://discovery.etcd.io/2059ff6f760766eab9a54335272a2e4b
root      1089  0.9  0.6  16112 13964 ?        Ssl  19:22   0:04 /etcd -name 59b8789c69504c34bb93c2c1b18c4765 -data-dir /var/lib/etcd2 -advertise-client-urls http://172.17.8.100:2379,http://172.17.8.100:4001 -listen-client-urls http://0.0.0.0:2379,http://0.0.0.0:4001 -initial-advertise-peer-urls http://172.17.8.100:2380,http://172.17.8.100:7001 -listen-peer-urls http://0.0.0.0:2380,http://0.0.0.0:7001 --heartbeat-interval 400 --election-timeout 2000 --discovery https://discovery.etcd.io/2059ff6f760766eab9a54335272a2e4b
core      1400  0.0  0.0   4288   272 pts/0    R+   19:30   0:00 grep --colour=auto etcd
core@deis-01 ~ $ cat /etc/os-release 
NAME=CoreOS
ID=coreos
VERSION=647.2.0
VERSION_ID=647.2.0
BUILD_ID=
PRETTY_NAME="CoreOS 647.2.0"
ANSI_COLOR="1;32"
HOME_URL="https://coreos.com/"
BUG_REPORT_URL="https://github.com/coreos/bugs/issues"
```